llvm/lib/Support/DXILABI.cpp

```diff
@@ -0,0 +1,20 @@
+//===-- DXILABI.cpp - ABI Sensitive Values for DXIL --------------*- C++-*-===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements functions that can be reused accross different stages
+// dxil generation.
+//
+//===----------------------------------------------------------------------===//
+
+#include "llvm/BinaryFormat/DXContainer.h"

```

Comment:
`Support` can't depend on `BinaryFormat`

Reply:
Looks like you removed the actual dependency but forgot to remove the include here